tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -1176,7 +1177,7 @@ String setLocales(WebContext ctx, int theirId, String theirEmail, String newLoca
 
         if (!intLocs) {
             newLocales = normalizeLocaleList(newLocales);
-            if (newLocales.isEmpty()) newLocales = "und";
+            if (newLocales.isEmpty()) newLocales = "mul";
```

Comment:
Ideally there might be a constant for the string literal "mul", referring to tr35 "The content contains more than one language or text that is simultaneously in multiple languages (such as brand names)."

Reply:
@btangmu this particular line is only for the user update - it's not related to "multiple languages" per se. Should it instead be instead a constant DEFAULT_LOCALE or something?

Reply:
Yes, maybe SpecialLocales.DEFAULT_LOCALE? Anything like that is an improvement for encapsulation

Reply:
removed.